### ISSUE TITLE
Input Format for MaskRCNN

### ISSUE THREAD

**Issue by natasasdj** [facebookresearch/Detectron#425]
Is COCO json annotation format the only input format supported by MaskRCNN (and other models in the detectron)? 
Is it possible in the detectron to train only with the part of the coco_train2014 data set without changing the annotation json file?

**Comment by ir413** [comment]
Yes, annotation ids should be unique across the annotations (not only per image_id).
